src/stockwatch/entities/transactions.py

```diff
@@ -111,3 +44,24 @@ class ShareTransaction:
     def transaction_date(self) -> date:
         "Transaction date"
         return self.transaction_datetime.date()
+
+
+@dataclass(frozen=True, order=True)
+class CashSettlement:
+    """For representing a cash settlement of a stock share delisting at a certain date.
+
+    Attributes
+    ----------
+    settlement_datetime   : the date  and time for which the value of the share position is registered
+    isin                  : the ISIN code / Symbol string used to identify a share
+    amount                : the value of the transaction in EUR
```

Comment:
Not sure if we should specify explicitly in eur, The nice thing about the `Amount` is that it could in theory be anythign.

Reply:
Yes, it could be any currency, but I want it to be in EUR (so apply exchange on the amount before creating the instance).
But you are right, I do not check on that. Shall I add an assert?

Reply:
I think it would be good to either have the check, or remove the comment. In principle the `CashSettlement` class does not really care what currency it is right? So to me it makes sense to just remove the comment here.

Reply:
You're right, done.